micropython/usb/examples/device/cdc_repl_example.py

```diff
@@ -0,0 +1,41 @@
+# MicroPython USB CDC REPL example
+#
+# Example demonstrating how to use os.dupterm() to provide the
+# MicroPython REPL on a dynamic CDCInterface() serial port.
+#
+# Note that if you run this example on the built-in USB CDC port via 'mpremote
+# run' then you'll have to reconnect after it re-enumerates, and it may be
+# necessary afterward to type Ctrl-B to exit the Raw REPL mode and resume the
+# interactive REPL back.
+#
+# This example uses the usb-device-cdc package for the CDCInterface class.
+# This can be installed with:
+#
+# mpremote mip install usb-device-cdc
```

Comment:
I suggest simple yet full instructions to run this example (I know it's described in the README but I think adding it to each example is useful); eg:
```
To run this example:
1. Make sure `usb-device-cdc` is installed via: mpremote mip install usb-device-cdc
2. Run the example via: mpremote run cdc_repl_example.py
```